src/_guides/language/spec.md

```diff
@@ -1,73 +1,33 @@
 ---
 layout: default
 title: "Dart Language Specification"
-description: "The Dart language specification and proposed changes."
+description: "The Dart language specification."
 ---
 
-Download the _Dart Programming Language Specification_ from
-the Ecma website:
-
-* <a href="http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-408.pdf"
-   target="_blank">Dart Programming Language Specification, 4<sup>th</sup> Edition</a>
-
+Use this page to find the formal Dart language specification.
 For a gentler introduction to the Dart language, see the
```

Comment:
"language" appears often in the opening paragraph, can we drop one occurrent to make things lighter?

> For a gentler introduction to the Dart, see ...